pkg/antctl/runtime/runtime.go

```diff
@@ -29,6 +29,7 @@ import (
 const (
 	ModeController     string = "controller"
 	ModeAgent          string = "agent"
+	ModeAgentWindows   string = "agent-windows"
```

Comment:
See comment above, I don't think we should define this constant here.
We should not conflate the FeatureGate "component" and the antctl runtime mode.